### PR TITLE
updated the grammar a little more

### PR DESCRIPTION
**Pull request by byronbytes**
and stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
-<h1>Discord Oxygen</h1>
```
Comment (coolman3289) [inline README.md:1]: Why remove this?

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Discord Oxygen is a universal client modification with many features and include
       <li>Animated Profile Banner</li>
       <li>Server Boosts (see Oxygen Boosts)</li>
       <li>Unlimited Upload Filesize: Normal 10MB, Discord Nitro 100MB, Discord Oxygen 100,000MB</li>
-      <li>Unlimited Servers: normal 100 Servers, Discord Nitro 200 Servers, Discord Oxygen ∞ Servers<br>(after more than 10,000 Servers your PC might get a bit laggy)</li>
+      <li>Unlimited Servers: normal 100 Servers, Discord Nitro 200 Servers, Discord Oxygen ∞ Servers<br>(after 10,000 servers your PC might lag)</li>
```
Comment (coolman3289) [inline README.md:33]: Should be left as “more than 10000”

File: README.md [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
 <h2>Important Information</h2>
 <p>
   Many features listed here are only client sided!<br>
   This means that some things like a custom discriminator or animated profile picture/banner are <strong>only visible to other Discord Oxygen users!</strong><br>
-  We try to make as many things globally visible as possible, but that's not possible for everything.<br>
+  We try to make as many things globally visible as possible, but that's unfortunately not possible for everything.<br>
   (For example the custom emojis are real emojis for Discord Oxygen-users, but for non Discord-Oxygen-users they only <strong>look like</strong> custom emojis.)<br>
   <strong>You won't get a real Discord Nitro subscription for free.</strong> Any tool claiming to do that is most likely a scam.<br>
   <br>
-  <strong>Important:</strong> Only use exploits on your own accounts or on accounts where you have the legal permission to do so. Hacking accounts is <strong>illegal</strong>, you will get in serious trouble if you don't have the permission. Please also consider that this wouldn't only be illegal but also unethical.
```
Comment (coolman3289) [inline README.md:15]: Think that should be left as accounts, as you may have more than 1 account.

**Review by coolman3289 — CHANGES_REQUESTED** [review]
Think some things changes need to be reverted.

File: README.md (hunk 1 above)
Comment (byronbytes) [reply README.md:1]: It makes sense to replace it with the logo instead, it's basically repeating the word twice.

File: README.md (hunk 2 above)
Comment (byronbytes) [reply README.md:33]: "after more than 10000" sounds gramatically incorrect
"after 10000" sounds better

File: README.md (hunk 3 above)
Comment (byronbytes) [reply README.md:15]: Will fix this as account(s)

**Pull request merged**